### PR TITLE
Metro Itinerary Change list of times to arrivals when arrive by is selected

### PR DESCRIPTION
**Pull request by miles-grant-ibigroup**
**Description:**

At the moment, the list of times in the metro itinerary row is departure times. This PR changes this to be arrival times when arrive by is selected.

**PR Checklist:**
- [X] Does the code follow accessibility standards (WCAG 2.1 AA Compliant)?
- [ ] Are all languages supported (Internationalization/Localization)?
- [ ] Are appropriate Typescript types implemented?

### CODE REVIEW

**Review by AdrianaCeric — APPROVED** [review]
Great work on this change!
The Arrive at aria-label is set to "Arrive" for both "You leave" and "You arrive" menu items. Do we want to keep doing that?
image

**Comment by miles-grant-ibigroup** [comment]
Good catch! This label is removed in another PR.

File: lib/components/narrative/metro/departure-times-list.tsx [hunk 1]
```diff
@@ -67,7 +73,13 @@ export const DepartureTimesList = (props: DepartureTimesProps): JSX.Element => {
                 : ''
             }`}
           >
-            <FormattedTime value={getFirstLegStartTime(time.legs)} />
+            <FormattedTime
```
Comment (binh-dam-ibigroup) [inline lib/components/narrative/metro/departure-times-list.tsx:76]: Was the conflict with the button title above handled between this PR and #789?

**Review by binh-dam-ibigroup — COMMENTED** [review]
Could you advise regarding https://github.com/opentripplanner/otp-react-redux/pull/818/files#r1132834280, please?

**Comment by miles-grant-ibigroup** [comment]
@binh-dam-ibigroup Merge is complete, all seems to be behaving correctly.

**Review by binh-dam-ibigroup — APPROVED** [review]
LGTM

**Pull request merged**